amy/trainings/views.py

```diff
@@ -134,31 +124,16 @@ def all_trainees(request):
     if request.method == "POST" and "discard" in request.POST:
         # Bulk discard progress of selected trainees
         form = BulkAddTrainingProgressForm()
```

Comment:
Should these 3 lines remain? We could probably leave the "elif" from L149 as "if" and skip the check for "discard" in POST since that form was removed.

Reply:
Good point. I'll make this change.